chia/wallet/wallet.py

```diff
@@ -48,6 +48,7 @@
 
 if TYPE_CHECKING:
     from chia.server.ws_connection import WSChiaConnection
+    from chia.wallet.wallet_state_manager import WalletStateManager
```

Comment:
@wallentx Coverage reports this line here which, this can be ignored.
@altendky Any way to not fail for type checking stuff like this? :)

Reply:
https://github.com/Chia-Network/chia-blockchain/pull/15473